### PR TITLE
Create a skeleton stage0 BIOS.

### PR DESCRIPTION
**Pull request by andrisaar**

### CODE REVIEW

File: stage0/layout.ld [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+HIDDEN(TOP = 4096M);
+HIDDEN(BIOS_SIZE = 256K);
```
Comment (conradgrobler) [inline stage0/layout.ld:18]: Is this intended to be different to the 64KiB specified in the readme?

**Review by conradgrobler — APPROVED** [review]
Nice

File: stage0/src/asm/boot.s [hunk 2]
```diff
@@ -0,0 +1,62 @@
+.code16
+.section .text16, "ax"
+.global _start
+_start :
+    # Enter log mode. This code is inspired by the approach shown at
```
Comment (tiziano88) [inline stage0/src/asm/boot.s:5]:
```suggestion
    # Enter long mode. This code is inspired by the approach shown at
```

**Review by tiziano88 — APPROVED** [review]

File: stage0/layout.ld (hunk 1 above)
Comment (andrisaar) [reply stage0/layout.ld:18]: Good catch, no. Once Rust came into play 64K wasn't enough any longer.

File: Cargo.toml [hunk 3]
```diff
@@ -70,6 +70,7 @@ members = [
 exclude = [
   "experimental/oak_baremetal_app_crosvm",
   "oak_functions/loader/fuzz",
+  "stage0",
```
Comment (tiziano88) [inline Cargo.toml:73]: oak_stage0

**Pull request merged**